code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -515,6 +505,13 @@
 	list_reagents = list("shamblers" = 30)
 	foodtype = SUGAR | JUNKFOOD
 
+/obj/item/reagent_containers/food/drinks/soda_cans/buzz_fuzz
+	name = "Buzz Fuzz"
+	desc = "The sister drink of Shambler's Juice! Uses real honey, making it a sweet tooth's dream drink. The slogan reads ''A Hive of Flavour'', there's also a lable about how its adddicting."
```

Comment:
label not lable, it's and not its.

Reply:
It is not It's

Reply:
it is it's. "it is", it's. "its" is the possessive form

Reply:
"There's also a label about how it is addicting" is the proper way to do it. Also you have three ds in "addicting", which I just noticed.